### DISCUSSION TITLE
Hide content of navigation column

### DISCUSSION THREAD

**Discussion by lenisko** [squidfunk/mkdocs-material#5891] [Ideas]
I'm aware of the ability to hide navigation bar completely (which would change width of columns), but is there a way to actually hide a content for e.g: small/short pages?

So "Ogólne" on the left would disapear, but would stay on the navigation bar at the top and columns would stay as they are.

![2023-08-24_21-22-42-firefox](https://github.com/squidfunk/mkdocs-material/assets/10072920/d69e964d-d1b6-410c-b508-698933d08117)

**Comment by squidfunk** [comment]
Yes, definitely possible. The easiest way is to add one line to the Markdown file of the page in question:

``` html
<style>
  @media screen and (min-width: 76.25em) {
    .md-sidebar--primary { visibility: hidden }
  }
</style>
```

**Reply by lenisko** [reply]
Thank you :)

**Reply by lenisko** [reply]
Actually... Hamburger/side menu is not working on mobile for sites with this addition 🤔

**Reply by squidfunk** [reply]
I adjusted the example.
